spark/v2.4/spark/src/test/java/org/apache/iceberg/spark/data/TestParquetAvroReader.java

```diff
@@ -116,8 +120,8 @@ public void testStructSchema() throws IOException {
         long end = System.currentTimeMillis();
         long duration = end - start;
 
-        System.err.println("XOR val: " + val);
-        System.err.println(String.format("Reassembled %d records in %d ms", count, duration));
+        LOG.error("XOR val: {}", val);
+        LOG.error("Reassembled {} records in {} ms", count, duration);
```

Comment:
This is ignored, so there is no need to update this. It is for sanity checking the implementation.

Reply:
Although this is ignored, but I also think this should be removed.